data/internal/Rogue_Mage/units/Bandit_Shadow_Lord.cfg

```diff
@@ -16,18 +16,34 @@
     cost=62
     {AMLA_DEFAULT}
     usage=mixed fighter
-    description= _ "The fruits of the paths of either light or darkness are well known to magi: the brilliance of a white mage or the deep gloom of a necromancer. Much less well known is the art of the Shadow Lord, those few who master a union of both white and black magic. Though decades of zealous study and self-neglect have left them drained of strength, these savants have transcended the need for crude physical weapons, instead channeling their energies into devastating magical attacks that undercut their enemies’ resistances."
-    die_sound={SOUND_LIST:HUMAN_DIE}
+    #po: I think of regular mages as modern-day professors, or perhaps ancient greek philosphers. In comparison, a Shadow Lord is more like an unemployed, obsessive savant who hasn't showered in three months but somehow discovered the secret to cold fusion while working alone in his dingy apartment.
```

Comment:
'philos**o**phers' was missed.

Also, would an apartment be a common thing in Wesnoth times? Particularly in the less populated areas a shadow mage would find refuge in.

Then again, I recognise this is just a po hint and a lot of this reads like from an American perspective to me.